src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-nested-ternary */
+import { Link, NavLink, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { SEX } from '../consts/Sex';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
```

Comment:
Fix all cases about boolean variables
```suggestion
  const [isError, setIsError] = useState(false);
```